shell/app/common/components/log/log-roller.scss

```diff
@@ -3,6 +3,10 @@ $log-bg-color: #3c444f;
 
 .log-viewer {
   height: 100%;
+
+  .ant-tabs-content {
```

Comment:
if we override `ant-tabs-content` in antd extension, would it cause problem?

Reply:
Maybe, set as global style have high risks.